templates/shared/pricing/_openstack-packages.html

```diff
@@ -18,27 +17,29 @@ <h2 class="p-heading--four">Private Cloud Build</h2>
       <li class="p-list__item is-ticked">Upgrades on demand</li>
       <li class="p-list__item is-ticked">High availability</li>
     </ul>
+    <p>Design and delivery on certified hardware.</p>
+    <p><a href="/openstack/consulting">Get OpenStack&nbsp;&rsaquo;</a></p>
   </div>
   <div class="col-{{ 12 // columns }} p-card">
     <h2 class="p-heading--four">Private Cloud Build Plus</h2>
-    <p><span class="p-heading--two">$150,000</span><br />Custom architecture for your specific industry or workload requirements.</p>
+    <p><span class="p-heading--two">$150,000</span> fixed price</p>
     <hr class="u-sv1" />
-    <p>Co-designed workshops to determine the optimal architecture for your workloads and integration requirements. Our engineers then deploy it for you.</p>
+    <p>Design and Deployment of a secure, carrier grade, feature rich OpenStack custom architecture</p>
```

Comment:
missing full-stop at the end
```suggestion
    <p>Design and Deployment of a secure, carrier grade, feature rich OpenStack custom architecture.</p>
```

Reply:
Fixed here https://github.com/canonical-web-and-design/ubuntu.com/pull/8129